hm5_1.py

```diff
@@ -0,0 +1,20 @@
+""" Task 1: Заменить символ “#” на символ “/” в строке:
+
+Строка 'www.my_site.com#about'
+
+"""
+
+# variant 1.
+
+print("www.my_site.com#about".replace("#", "/"))
+
+
+# Variant 2.
+
+TXT_11 = "www.my_site.com#about"
+
+LST = list(TXT_11)
+X = LST.index("#")
```

Comment:
why you can not use simple 'find' method?
https://www.w3schools.com/python/ref_string_find.asp

Reply:
Understand your message.